shell.nix

```diff
@@ -0,0 +1,3 @@
+{ system ? builtins.currentSystem }:
```

Comment:
when is this useful? shouldn't `nix develop` be used instead?

Reply:
Its a silly trick that lets you use direnv with the flake.

Reply:
Ok makes sense.